optimizely/project_config.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, datafile, logger, error_handler):
         """
 
         config = json.loads(datafile)
-        self._datafile = datafile
+        self._datafile = str(datafile)
```

Comment:
This is per @mjc1283 's request. That's fine w me if we want to return a string object.